src/views/RecIM/views/Activity/index.jsx

```diff
@@ -288,6 +313,49 @@ const Activity = () => {
                 activity={activity}
               />
             )}
+            {openSettings && (
+              <GordonDialogBox
+                title="Admin Settings"
+                fullWidth
+                open={openSettings}
+                cancelButtonClicked={() => setOpenSettings(false)}
+                cancelButtonName="Close"
+              >
+                <br />
+                <Grid container alignItems="center" justifyContent="space-between">
+                  <Grid item>
+                    <Typography>Permanently delete the activity '{activity.Name}'</Typography>
```

Comment:
I think I would prefer for there to be a bit more information on the Activity. Especially a check for if any games/series have started. It's a bit nitpicky so I would consider it non-blocking and we can obviously add it later.

Reply:
lol ok 👍🏼